src/mediaelement.js

```diff
@@ -105,6 +106,14 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
         this.seekTo(start);
         this.media.play();
         end && this.setPlayEnd(end);
+        var my = this;
+        function updateProgress(){
+            my.fireEvent('audioprocess', my.getCurrentTime());
+            if (my.updateProgressAnimating) {
+                requestAnimationFrame(updateProgress);
+            }
+        }
+        this.updateProgressAnimating = requestAnimationFrame(updateProgress);
```

Comment:
Sorry, not in the drawer code, but in `wavesurfer.js` (since it has access to both the backend and the drawer).